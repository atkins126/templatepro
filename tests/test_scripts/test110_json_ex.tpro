{{:json2}}

{{if (json2)}}OK{{else}}KO{{endif}}
{{if (!json2)}}OK{{else}}KO{{endif}}


{{loop(json2.array) as j}}
  {{:j.value}}
  {{:j.subarray}}
{{endloop}}