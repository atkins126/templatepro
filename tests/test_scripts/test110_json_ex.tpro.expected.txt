{"foo":"bar","bar":"foo","array":[10,20,{"value":12.3,"subarray":[100,200]}]}

KO

  
  
  
  
  12.3
  [100,200]
